ethereum/jsonrpc/src/main/java/tech/pegasys/pantheon/ethereum/jsonrpc/internal/methods/privacy/EeaGetPrivateTransaction.java

```diff
@@ -60,7 +60,11 @@ public JsonRpcResponse response(final JsonRpcRequest request) {
     final String enclaveKey = parameters.required(request.getParams(), 0, String.class);
     try {
       ReceiveResponse receiveResponse =
-          getReceiveResponseFromEnclave(enclaveKey, privacyParameters.getEnclavePublicKey());
+          getReceiveResponseFromEnclave(
+              Base64.getEncoder()
```

Comment:
I believe this could benefit from the changes in https://github.com/PegaSysEng/pantheon/pull/1667?

`public static String asBase64String(final BytesValue bytesValue)`